config/resources/users-domain.lisp

```diff
@@ -12,6 +12,9 @@
                                         :as "account")
              (organization              :via      ,(s-prefix "org:memberOf")
                                         :as "organization")
+             (sign-flow                 :via ,(s-prefix "dct:creator")
+                                        :inverse t
+                                        :as "sign-flow")
              ) ;; Has one relation adms:identifier to type adms::Identifier created in user-management-service
```

Comment:
Mogelijk willen we deze relatie niet expliciet inverse leggen hier en zo het gebruik van inverse relatie in de frontend afblocken
Op zich is het voldoende dat de sign-flow een relate heeft naar zijn user.
net zoals een agendaitem of een subcase ook een relatie heeft naar een user, maar die relatie staat hier niet bij.

reden daarvoor is dat we willen vermijden dat er aan 1 user een hele lijst hangt die we moeten updaten, of voorkomen dat we een save doen met maar een halve lijst (ember aggressive save), en daardoor een deel triples uit de database gooien.
Zelf als je ooit de intentie hebt om in de frontend alle sign-flows op te halen van een bepaalde gebruiker, zou ik dit eruit laten. dan werken we best met een query om die data op te halen